WordPress/src/main/res/values/dimens.xml

```diff
@@ -17,7 +17,9 @@
     <dimen name="fab_margin_tablet">24dp</dimen>
     <dimen name="design_fab_size">56dp</dimen>
 
-    <item format="float" name="primary_button_disabled_alpha" type="dimen">0.12</item>
+    <item name="primary_button_disabled_alpha" format="float" type="dimen">@dimen/disabled_alpha
+    </item>
```

Comment:
Maybe I remember this is the way AS auto format this sort of things and this is less than a np, it's more a proof that I went into the code 😬! I would align on a single line, but really super minor so feel free to just ignore this!

Reply:
I switched formatting to multiline, otherwise AS will reformat it again in te future :)